content/kubermatic/master/installation/start_kkp/guides/setup_repository/_index.en.md

```diff
@@ -0,0 +1,220 @@
++++
+title = "Setup your Git repository"
+weight = 30
+enableToc = true
++++
+
+The goal of the setup is to put the downloaded directory structure into your Git repository,
+so let’s get started with creating a fresh repository for this purpose and then setup the required Variables or Secrets before
+pushing the code to the repository.
+
+{{% notice warning %}}
+Following sections will be different based on the combination of Git and Cloud providers you have chosen.
+{{% /notice %}}
+
+## Create Git repository
+
+{{< tabs name="Create Git repository" >}}
+{{% tab name="GitHub" %}}
+Create a new repository on GitHub [manually](https://docs.github.com/en/get-started/quickstart/create-a-repo) or using [GitHub CLI](https://cli.github.com/manual/gh_repo_create).
+
+Also prepare an [Access token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+for GitHub which will be used in the next section while preparing Secrets. Token will be used for the Flux bootstrap on repository.
+{{% /tab %}}
+{{% tab name="GitLab" %}}
+Create new repository on GitLab [manually](https://docs.gitlab.com/ee/user/project/working_with_projects.html#create-a-project).
+
+Also prepare a GitLab API token which will be used for GitOps setup.
+{{% /tab %}}
+{{% /tabs %}}
+
+
+## Prepare Cloud provider credentials
+
+{{< tabs name="Cloud Provider Credentials" >}}
+{{% tab name="AWS" %}}
+Login to [AWS console](https://console.aws.amazon.com/console) and create your access keys under IAM or using
+AWS CLI `aws iam create-access-key`.
+
+Credentials should be static and do not utilize any tools like `aws-iam-authenticator` because they are also stored as secret in your Kubernetes cluster.
+
+{{% /tab %}}
+{{% tab name="Azure" %}}
+Login to [Azure portal](https://portal.azure.com/) and create a role and service account (application).
+
+See [KubeOne documentation](https://docs.kubermatic.com/kubeone/master/architecture/requirements/machine_controller/azure/) for more details.
```

Comment:
We should use references instead of a direct link for KubeOne docs.

Reply:
done